src/EditorFeatures/Core.Wpf/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -40,10 +40,11 @@ internal partial class SymbolSearchUpdateEngine
         private const string MicrosoftAssemblyReferencesName = "MicrosoftAssemblyReferences";
 
         /// <summary>
-        /// Cancellation support for the task we use to keep the local database up to date.
-        /// When VS shuts down it will dispose us.  We'll cancel the task at that point.
+        /// Cancellation support for the task we use to keep the local database up to date. Used
+        /// only in tests so we can shutdown gracefully.  In normal VS+OOP scenarios we don't care
+        /// about this and we just get torn down when the OOP process goes down.
         /// </summary>
-        private readonly CancellationToken _updateCancellationToken;
+        private readonly CancellationToken _testCancellationToken;
```

Comment:
❓ Why not remove this and pass it directly to `UpdateInBackgroundAsync()`?

Reply:
let me try that!

Reply:
ok. done.  thanks!